server.go

```diff
@@ -452,6 +452,11 @@ func (c *serverConn) run(sctx context.Context) {
 			if err != nil && err != io.EOF {
 				logrus.WithError(err).Error("error receiving message")
 			}
+			if err == io.EOF || err == io.ErrUnexpectedEOF {
```

Comment:
The `err == io.ErrUnexpectedEO` won't work unless you ignore it above.

Reply:
It would not have worked if there was a return statement in the if above, but there isn't. It would still be better to not log the `ErrUnecpectedEOF` though so we have created a new pr: #38. Pls, take a look.